doc/kubernetes/modules/ROOT/pages/running/metrics/jgrp_metrics.adoc

```diff
@@ -0,0 +1,198 @@
+= Network Metrics
+
+:description: This document contains details of the JGroups metrics that can be used to monitor your {jdgserver_name} deployment's performance.
+
+include::partial$running/infinispan-attributes.adoc[]
+
+{description}
+
+====
+*Global tags*
+
+* `cluster=<name>`: The {jdgserver_name} creates 2 cluster, one for the intra-cluster communication and another for cross-site communication.
+This tag identify to which cluster the metric refers to.
+* `node=<node>`: The name of the node reporting the metric.
+====
+
+== Monitoring Response Time
+
+{jdgserver_name} computes the responses time for all the request sent (it includes processing time).
+The response time rate should be stable through the cluster lifecycle.
+
+TIP: An increase in response time may indicate a degraded cluster.
+
+|===
+|Metric |Description
+
+|`vendor_jgroups_stats_sync_requests_seconds_count`
+| The number of synchronous requests to a receiver node.
+
+|`vendor_jgroups_stats_sync_requests_seconds_sum`
+| The total duration of synchronous request to a receiver node
+
+|===
+
+====
+*Tags*
+
+* `target_node=<node>`: This tag identify the receiver node.
+====
+
+NOTE: {histogram-note}
+
+== Monitoring Bandwidth
+
+JGroups reports all the bytes sent and receive by a node.
+The following metrics help to monitor the traffic, and the bandwidth used by the cluster.
+
+|===
+|Metric |Description
+
+| `vendor_jgroups_tcp_get_num_bytes_received`
+| The total number of bytes received by a node (intra-cluster communication).
+
+| `vendor_jgroups_tcp_get_num_bytes_sent`
+| The total number of bytes sent by a node (intra-cluster communication).
+
+| `vendor_jgroups_tunnel_get_num_bytes_received`
+| The total number of bytes received by a node (cross-site communication).
+
+| `vendor_jgroups_tunnel_get_num_bytes_sent`
+| The total number of bytes sent by a node (cross-site communication).
+
+|===
+
+== Thread Pool
+
+When a message is received from another node, it is sent to a thread pool to be delivered to {jdgserver_name}.
+If no threads are available, the message is dropped, leading to retransmission and increase network usage.
+
+The metrics below report the thread pool size.
+
+TIP: In a healthy cluster, the thread pool should never be closer to its maximum size (by default, `200` threads).
+
+NOTE: Thread pool metrics are not available with virtual threads.
+
+|===
+|Metric |Description
+
+|`vendor_jgroups_tcp_get_thread_pool_size`
+|Current number of threads in the thread pool, for the intra-cluster communication.
+
+|`vendor_jgroups_tunnel_get_thread_pool_size`
+|Current number of threads in the thread pool, for the cross-site communication.
+
+|===
+
+== Flow Control
+
+Flow control takes care of adjusting the rate of a message sender to the rate of the slowest receiver over time.
+This is implemented through a credit based system, where each sender credits and decrements them whenever a message is sent.
```

Comment:
I don't understand this. When does the number of credits increase and when does it decrease?